contracts/src/v0.8/automation/upkeeps/LinkAvailableBalanceMonitor.sol

```diff
@@ -128,32 +161,32 @@ contract LinkAvailableBalanceMonitor is ConfirmedOwner, Pausable, KeeperCompatib
   }
 
   /**
-   * @notice Gets a list of proxies that are underfunded, up to the MAX_PERFORM size
+   * @notice Gets a list of proxies that are underfunded, up to the s_maxPerform size
    * @dev the function starts at a random index in the list to avoid biasing the first
    * addresses in the list over latter ones.
-   * @dev the function will check at most MAX_CHECK proxies in a single call
-   * @dev the function returns a list with a max length of MAX_PERFORM
+   * @dev the function will check at most s_maxCheck proxies in a single call
+   * @dev the function returns a list with a max length of s_maxPerform
    * @return list of target addresses which are underfunded
    */
   function sampleUnderfundedAddresses() public view returns (address[] memory) {
     uint256 numTargets = s_watchList.length();
     uint256 numChecked = 0;
     uint256 idx = uint256(blockhash(block.number - 1)) % numTargets; // start at random index, to distribute load
-    uint256 numToCheck = numTargets < MAX_CHECK ? numTargets : MAX_CHECK;
+    uint256 numToCheck = numTargets < s_maxCheck ? numTargets : s_maxCheck;
     uint256 numFound = 0;
-    address[] memory targetsToFund = new address[](MAX_PERFORM);
+    address[] memory targetsToFund = new address[](s_maxPerform);
     for (; numChecked < numToCheck; (idx, numChecked) = ((idx + 1) % numTargets, numChecked + 1)) {
       (address target, uint256 minBalance) = s_watchList.at(idx);
       (bool needsFunding, ) = _needsFunding(target, minBalance);
       if (needsFunding) {
         targetsToFund[numFound] = target;
         numFound++;
-        if (numFound == MAX_PERFORM) {
+        if (numFound == s_maxPerform) {
```

Comment:
since this is now read multiple times, let's assign it to a memory variable instead of reading from storage multiple times

Reply:
Done!